lib/oeqa/runtime/cases/nnstreamer.py

```diff
@@ -32,6 +32,20 @@ def test_nnstreamer(self):
         msg = " NNSTREAMER UNITTEST FAILED ( unittest_src_iio ): %s" % output
         self.assertEqual(status, 0, msg=msg)
 
+        cmd = 'export LD_LIBRARY_PATH=/usr/lib/gstreamer-1.0:$LD_LIBRARY_PATH; ' \
+        'export NNSTREAMER_SOURCE_ROOT_PATH=/usr/lib/nnstreamer/unittest;' \
+        '/usr/lib/nnstreamer/unittest/unittest_tizen_tensorflow_lite --gst-plugin-path=/usr/lib/gstreamer-1.0'
```

Comment:
```suggestion
        cmd = 'LD_LIBRARY_PATH=/usr/lib/gstreamer-1.0:$LD_LIBRARY_PATH ' \
        'NNSTREAMER_SOURCE_ROOT_PATH=/usr/lib/nnstreamer/unittest ' \
        '/usr/lib/nnstreamer/unittest/unittest_tizen_tensorflow_lite --gst-plugin-path=/usr/lib/gstreamer-1.0'
```

How about changing it not to use ```export```?

Reply:
Not to use `export` seems good. I updated the commit.